sdks/python/tox.ini

```diff
@@ -343,8 +343,7 @@ commands =
   /bin/sh -c "pip freeze | grep -E onnx"
   # Run all ONNX unit tests
   pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_onnx {posargs}
-  
-[testenv:py{37,38,39,310}-tf-{211}]
+
 [testenv:py{37,38,39,310}-tensorflow-{29,210,211}]
```

Comment:
Hm, looks like all tf tests are currently being skipped - https://ci-beam.apache.org/job/beam_PostCommit_Python38_PR/697/testReport/apache_beam.ml.inference.tensorflow_inference_it_test/TensorflowInference/

I think to get these to run correctly, we should add `tensorflow` to our test dependencies - https://github.com/apache/beam/blob/f358fcdc8c8a844e1bdfc8ca8f6fe886734c8df9/sdks/python/setup.py#L288

It actually looks like we have the same problem with pytorch and onnx, but that could be a follow up PR (from you or someone else)

Reply:
Oooh.. Thanks for catching that

Reply:
Created https://github.com/apache/beam/issues/25547 to track pytorch and onnx

Reply:
Thanks - I have a draft PR for it (https://github.com/apache/beam/pull/25548)